src/swagger/routes/misc.yaml

```diff
@@ -0,0 +1,71 @@
+/health:
+  get:
+    tags:
+      - Miscellaneous
+    summary: Health of the service
+    responses:
+      200:
+        description: Success
+        content:
+          application/json:
+            schema:
+              $ref: '../components/schemas/health.yaml'
+      500:
+        description: Health Check Failed
+/features:
```

Comment:
The router transform is internal. I wouldn't document this route.

Reply:
We are planning to integrate swagger for internal team ease of use (one the major reasons). Secondary is for external documentation as well.